### PR TITLE
[12.0][MIG] Migrated partner_prospect to v12

### PR DESCRIPTION
**Pull request by sudhir-erpharbor**
Please review and provide feedback.

### CODE REVIEW

File: partner_prospect/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    'child_ids.invoice_ids'))
+            partner.prospect = (
+                not sale_ids.filtered(
+                    lambda r: r.state not in
+                    ('draft', 'sent', 'cancel')) and
+                not invoice_ids.filtered(
+                    lambda r: r.type in ('out_invoice', 'out_refund')))
+
+    prospect = fields.Boolean(
+        string='Prospect', compute='_compute_prospect', default=False,
```
Comment (nikul-serpentcs) [inline partner_prospect/models/res_partner.py]: @sudhir-erpharbor IMO `Boolean` is always False, No need to `default=False`

File: partner_prospect/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```
Comment (nikul-serpentcs) [inline partner_prospect/models/res_partner.py:1]: @sudhir-erpharbor Add `Copyright`

File: partner_prospect/tests/test_partner_prospect.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```
Comment (nikul-serpentcs) [inline partner_prospect/tests/test_partner_prospect.py:1]: @sudhir-erpharbor Add `Copyright`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Review by rven — CHANGES_REQUESTED** [review]
Hi @sudhir-erpharbor,

As written in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

Use new README by fragment system, copying https://github.com/OCA/maintainer-tools/tree/master/template/module/readme and editing or removing the needed sections. You can then run locally oca-gen-addon-readme if you have OCA/maintainer-tools installed and you want to pregenerate the README.rst file (preferred), or keep previous README.rst file.

**Comment by sudhir-erpharbor** [comment]
Done. Please review.

**Comment by rcastro-tyc** [comment]
Hi @pedrobaeza, Yes, I reviewed it before making the PR, I decided to do it again only because of the date the PR was made and because I needed it for a database.

Regards,

File: partner_prospect/models/res_partner.py (hunk 2 above)
Comment (sudhir-erpharbor) [reply partner_prospect/models/res_partner.py:1]: There were not any copyright in v11. What should it be exactly?

**Review by rven — APPROVED** [review]

File: partner_prospect/models/res_partner.py (hunk 2 above)
Comment (nikul-serpentcs) [reply partner_prospect/models/res_partner.py:1]: In V8 Copyright here https://github.com/OCA/sale-workflow/blame/8.0/partner_prospect/models/res_partner.py#L2

But new version not found anywhere, keep as it is, And What Do you think about this senior?

**Review by jimhoefnagels — APPROVED** [review]

**Comment by randall-vx** [comment]
Hi ! Any update about this PR ? Regards,

**Comment by randall-vx** [comment]
Hi @pedrobaeza, any update to this PR ?

**Pull request merged**